src/runtime_src/core/tools/common/XBMain.cpp

```diff
@@ -98,6 +100,11 @@ void  main_(int argc, char** argv,
     XBU::report_commands_help(_executable, _description, globalOptions, hiddenOptions, _subCmds);
     return;
   }
+
+  if(bVersion) {
+    XBU::pretty_print_xrt_version();

```

Comment:
Ideally, it would be great to do the following:
`std::cout << XBU::pretty_print_xrt_version();`

Instead of relying on the "default" `std::cout` to be used in the calling function.

Could you refactor the code to support this use-case. 

Note
It might be best to change the signature of `XBU:pretty_print_xrt_version()` to:
`std::string get_xrt_pretty_version();`